include/internal/ktls.h

```diff
@@ -222,6 +222,11 @@ static ossl_inline ossl_ssize_t ktls_sendfile(int s, int fd, off_t off,
 #    define OPENSSL_KTLS_TLS13
 #    if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 2, 0)
 #     define OPENSSL_KTLS_AES_CCM_128
+#     if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 11, 0)
+#      ifndef OPENSSL_NO_CHACHA
```

Comment:
Hmmm....actually....do we need this guard at all? In a no-chacha build, does anything actually break without this guard?

Reply:
Currently we have plenty of OPENSSL_NO_CHACHA ifndefs in the libssl code, so I do not think it will be particularly harmful to add this one as well even though it might not be needed.

Reply:
Also, note that TLS 1.3 only offloads TX to the kernel currently, so you will still need the EVP to decrypt on receive even when KTLS is enabled.